int32              object_count
TrackTarget[250]   track_objects